File: LeetCode/cpp/2114_Maximum_Number_of_Words_Found_in_Sentences.cpp
class Solution {
public:
	int mostWordsFound(vector<string>& sentences) {
		int answer = -1;

		for (int i = 0; i < sentences.size(); i++) {
			int cnt = 1;

			for (int j = 0; j < sentences[i].size(); j++) {
				if (sentences[i][j] == ' ') {
					cnt++;
				}
			}

			answer = max(answer, cnt);
		}

		return answer;
	}
};

File: LeetCode/java/2114_Maximum_Number_of_Words_Found_in_Sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int answer = -1;

        for (int i = 0; i < sentences.length; i++) {
            int cnt = 1;

            for (int j = 0; j < sentences[i].length(); j++) {
                if (sentences[i].charAt(j) == ' ') {
                    cnt++;
                }
            }

            answer = Math.max(answer, cnt);
        }

        return answer;
    }
}